src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts

```diff
@@ -16,7 +16,8 @@ import {
 import * as fs from "fs";
 import fetch from "node-fetch";
 import { MimeTypes } from "./mc-fus-mime-types";
-const Path = require("path");
+import * as Path from "path";
+import "abort-controller/polyfill";
```

Comment:
Use regular import, not polyfill

Reply:
Somehow it doesn't work that way. I have no context on this fix so dropping it as is for now to prioritize other work.